src/components/Rockets.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { reserveRocket, cancelReservation, fetchRockets } from '../redux/rocketActions';
+import '../rockets.css';
+
+const Rockets = () => {
+  const dispatch = useDispatch();
+  const rocketData = useSelector((state) => state.rockets);
+
+  useEffect(() => {
+    if (rocketData.length === 0) {
+      fetch('https://api.spacexdata.com/v3/rockets')
+        .then((response) => response.json())
+        .then((data) => {
+          dispatch(fetchRockets(data));
+        })
```

Comment:
- [ ] Kindly filter the rocket data from the API before dispatching it to the store, based on [this requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store).

  ![image](https://github.com/lily4178993/space-traveller/assets/54848363/c6728052-0d46-41b5-84da-bbaf18832cd6)